src/Sarif.Converters/AndroidStudioConverter.cs

```diff
@@ -11,7 +11,7 @@
 using System.Xml;
 using Microsoft.CodeAnalysis.Sarif.Writers;
 
-namespace Microsoft.CodeAnalysis.Sarif.Converters
+namespace Microsoft.CodeAnalysis.Sarif.Converters 
```

Comment:
Looks like you accidentally added a trailing space on this line.

Reply:
I don't believe AppVeyor fires a build unless you change a code file.  Config files and scripts don't fire the CI build last I checked.